src/main/resources/static/codegen/built-cas2-api-spec.yml

```diff
@@ -3797,12 +3797,34 @@ components:
         - cas3_person_arrived
         - cas3_person_departed
         - application_timeline_note
+        - cas2_application_submitted
```

Comment:
re-using the enum here as it seems cas3 has done the same thing, an not sure we need our own namespaced enum 🤔